packages/lesswrong/components/localGroups/CommunityMap.tsx

```diff
@@ -54,7 +55,7 @@ const styles = createStyles((theme: ThemeType): JssStyles => ({
   mapButtons: {
     alignItems: "flex-end",
     position: "absolute",
-    top: 10,
+    top: isFriendlyUI ? 40 : 10,
```

Comment:
Just checking - I don't think we use the "map buttons" (legend) anywhere on the EA Forum? Where does WU use it?

Reply:
A new friendly UI forum would by default get this community view: https://danceforum-production.up.railway.app/community with filter buttons being there 

waking up removed that, but these are from my experience of setting up another new instance of the forum (west coast swing dance focused, still wip)

Reply:
Oh I see, got it